data_discovery_ai/test_pipeline.py

```diff
@@ -1,24 +1,4 @@
-from data_discovery_ai.pipeline import KeywordClassifierPipeline
-
-
-def pipeline(isDataChanged, usePretrainedModel, description, selected_model):
-    keyword_classifier_pipeline = KeywordClassifierPipeline(
-        isDataChanged=isDataChanged,
-        usePretrainedModel=usePretrainedModel,
-        model_name=selected_model,
-    )
-    if keyword_classifier_pipeline.usePretrainedModel:
-        keyword_classifier_pipeline.make_prediction(description)
-    else:
-        if keyword_classifier_pipeline.isDataChanged:
-            raw_data = keyword_classifier_pipeline.fetch_raw_data()
-            sampleSet = keyword_classifier_pipeline.prepare_sampleSet(raw_data=raw_data)
-        else:
-            sampleSet = preprocessor.load_from_file("keyword_sample.pkl")
-        train_test_data = keyword_classifier_pipeline.prepare_train_test_sets(sampleSet)
-        keyword_classifier_pipeline.train_evaluate_model(train_test_data)
-
-        keyword_classifier_pipeline.make_prediction(description)
+from data_discovery_ai.pipeline import pipeline
 
 
 def test():
```

Comment:
test stuff should be organised in `test` package instead of root rlevel of `data_discovery_ai` package, this is already `test` package that comply with poetry project structure.